packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -1,37 +1,48 @@
-import { Component, Element, Host, Prop, State, h } from '@stencil/core';
-import { isNumeric } from '../../utils/utils';
-import ascendingIcon from '../../assets/sort-arrow-up.svg?format=text';
-import descendingIcon from '../../assets/sort-arrow-down.svg?format=text';
-import { quicksort, reverseQuicksort } from '../../utils/dom-sort';
+/* eslint-disable i18next/no-literal-string */
+import { 
+  Component,
+  Host, 
+  Element,
+  h,
+  State,
+  Prop
+} from '@stencil/core';
 
 /**
- * This component expects `<va-table-row>` elements as children.
- * Children of each row element should be `<span>` elements. Table
- * semantics will be added and numeric columns will be right aligned.
- */
-
-/**
- * @componentName Table
- * @maturityCategory use
- * @maturityLevel best_practice
+ * This is a wrapper component whose job is to marshal the cells in slotted va-table-rows 
+ * through DOM manipulation for projection into the slots of the va-table-inner component for rendering
  */
 
 @Component({
   tag: 'va-table',
-  styleUrl: 'va-table.css',
+  styleUrl: 'va-table.scss',
   shadow: true,
 })
 export class VaTable {
-  headers: HTMLElement[] = null;
-
   @Element() el: HTMLElement;
 
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false;
+  
   /**
    * The title of the table
    */
-  @Prop() tableTitle: string;
+  @Prop() tableTitle?: string;
 
   /**
+   * UNCOMMENT when we add more variations
+   * If uswds is true, the type of table
+   */
+  // @Prop() tableType?:
+  //   | 'borderless'
+  //   | 'striped' = 'borderless';
```

Comment:
Instead of commenting out code blocks, could you just remove `striped` and have  `borderless` as the only option on the prop for now? Or does Stencil complain?

```
@Prop() tableType?:  | 'borderless';
```

Reply:
@jamigibbs - no problem, done!